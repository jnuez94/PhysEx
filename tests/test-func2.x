int i;
void func finishing_move(int i){
	if(i == 0){
		print("Kaahhh..meehhhhh...");
	}
	if(i == 1){
		print("haa..meehhhhh...");
	}
	else{
		print("HAAAAAAAAAAAAA!");
	}

}
void func main() {
	for(i = 0; i < 3; i = i + 1){
		finishing_move(i);
	}
}
