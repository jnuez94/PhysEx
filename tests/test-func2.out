Kaahhh..meehhhhh...
haa..meehhhhh...
HAAAAAAAAAAAAA!
